packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1006,7 +1006,7 @@ class RenderEditable extends RenderBox {
   @override
   double computeMaxIntrinsicWidth(double height) {
     _layoutText(double.infinity);
-    return _textPainter.maxIntrinsicWidth;
+    return _textPainter.maxIntrinsicWidth + cursorWidth;
```

Comment:
We should probably `assert(cursorWidth != null && cursorWidth >= 0.0)` in the RenderEditable constructor. There are many other places in the code that appear to make this assumption.  Not sure if cursorWidth == 0.0 really works; if not assert that it's > 0.0.

Reply:
It looks like cursorWidth 0.0 works and results in not showing the cursor at all, so I'm using `>=` in the assertion.